### PR TITLE
add structurdata/upfdata appendices to mandi

### PR DESCRIPTION
**Pull request by ltalirz**
was added to sintef tutorial; now forwarding to mandi

### CODE REVIEW

**Comment by ltalirz** [comment]
@sphuber moved these tutorials over from aiida-core (PR #169 added this, retrospectively, to the sintef tutorial).

This PR adds them also for the Mandi tutorial, so that this material can actually be used and we keep the material going forward.

File: docs/pages/2019_IIT_Mandi_India/sections/verdi_shell.rst [hunk 1]
```diff
@@ -379,6 +380,8 @@ Finally, you can list all the pseudo families present in the database with
 
     verdi data upf listfamilies
 
+A more in depth tutorial about working with `UpfData` nodes and pseudo potenial families can be found in :ref:`this appendix<2019_mandi_appendix_upf_data>`.
```
Comment (CasperWA) [inline docs/pages/2019_IIT_Mandi_India/sections/verdi_shell.rst]:
```suggestion
A more in-depth tutorial about working with ``UpfData`` nodes and pseudopotential families can be found in :ref:`this appendix<2019_mandi_appendix_upf_data>`.
```

File: docs/pages/2019_IIT_Mandi_India/sections/verdi_shell.rst [hunk 2]
```diff
@@ -271,6 +271,7 @@ Finally, we can also import the silicon structure from an external (online) repo
         print("Unit cell volume: ", structure.get_cell_volume())
 
 In that case two duplicate structures are found for 'Si'.
+A more in depth tutorial can be found in :ref:`this appendix<2019_mandi_appendix_structure_data>`.
```
Comment (CasperWA) [inline docs/pages/2019_IIT_Mandi_India/sections/verdi_shell.rst]:
```suggestion
A more in-depth tutorial can be found in :ref:`this appendix<2019_mandi_appendix_structure_data>`.
```

**Review by CasperWA — CHANGES_REQUESTED** [review]
Thanks @ltalirz!
I have made some suggested changes, which I will simply commit and then merge this in.

**Pull request merged**